torch/csrc/jit/tensorexpr/tensor.h

```diff
@@ -300,6 +307,20 @@ inline ExprHandle Placeholder::load(const std::vector<T>& args) const {
       new Load(data(), ExprHandleVectorToExprVector(params), new IntImm(1)));
 }
 
+template <typename... Ts>
+inline ExprHandle BufHandle::load(const Ts&... ts) const {
```

Comment:
Is it possible to define these in expr.h instead?  Putting them in tensor.h feels pretty random.

Reply:
Good point - I just pattern matched since the definitions for placeholder and tensor are both here.

Reply:
There are some annoyances here related to cyclic dependencies, since this definition of Load requires definitions from `ir.h`, which depends on `expr.h`. Specifically, `IntImm` and `ExprHandleVectorToExprVector`.

Reply:
I kinda don't like using templates in general as they tend to lead to more code being in header files and then we get these dependencies. But at the same time I find this particular API convenient. So idk :)
If we decide to use it everywhere, or kill it everywhere, we should keep #52777 in mind, and maybe restructure our header files once we're done.